playbooks/files/rax-maas/plugins/maas_common.py

```diff
@@ -39,8 +39,8 @@
                 'OS_ENDPOINT_TYPE': 'PublicURL',
                 'OS_IDENTITY_API_VERSION': None,
                 'OS_API_INSECURE': False,
-                'OS_VOLUME_API_VERSION': None,
-                'OS_IMAGE_API_VERSION': None, }
+                'OS_VOLUME_API_VERSION': 1,
+                'OS_IMAGE_API_VERSION': 1, }
```

Comment:
trailing `,` should be removed

Reply:
was just leaving it as it was before the change.  didn't know if someone left it in for some weird bug or not.  I can change.

Reply:
removed